SparkBuilderGeneratorPlugin/src/com/helospark/spark/builder/handlers/codegenerator/builderfieldcollector/ClassFieldCollector.java

```diff
@@ -39,7 +39,12 @@ public ClassFieldCollector(FieldNameToBuilderFieldNameConverter fieldNameToBuild
         this.applicableFieldVisibilityFilter = applicableFieldVisibilityFilter;
     }
 
-    public List<? extends BuilderField> findBuilderFieldsRecursively(TypeDeclaration originalOwnerClasss, TypeDeclaration currentOwnerClass) {
+    @Override
+    public List<? extends BuilderField> collectFields(TypeDeclaration typeDeclaration) {
+        return findBuilderFieldsRecursively(typeDeclaration, typeDeclaration);
+    }
+
+    private List<? extends BuilderField> findBuilderFieldsRecursively(TypeDeclaration originalOwnerClasss, TypeDeclaration currentOwnerClass) {
```

Comment:
parentOwnerClass or typo (sss)

Reply:
Thanks.
On closer inspection, it turned out to be removable completely. :smile: